onnxruntime/core/providers/js/operators/pool.h

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/providers/js/js_kernel.h"
+#include "core/providers/cpu/nn/pool_base.h"
+
+namespace onnxruntime {
+namespace js {
+
+#define POOL_ATTRIBUTES_JS_OBJ_MAPPING ({ \
+    "format": $15 ? "NHWC" : "NCHW",      \
+    "auto_pad": $1,                       \
+    "ceil_mode": $2,                      \
+    "count_include_pad": $3,              \
+    "storage_order": $4,                  \
+    "dilations": [$5, $6],                \
+    "kernel_shape": [$7, $8],             \
+    "pads": [$9, $10, $11, $12],          \
+    "strides": [$13, $14]                 \
```

Comment:
Not clear how things like '$15' are working given this is C++ code, or what the $x values relate to, to know why say the first line uses $15 and the second line $1.